src/db/migration/env.py

```diff
@@ -1,23 +1,20 @@
 import os
 from logging.config import fileConfig
 
-from dotenv import load_dotenv
-from sqlalchemy import engine_from_config
-from sqlalchemy import pool
-
 from alembic import context
-
-import src.db.models
```

Comment:
This import is used for alembic to find the db models. I suggest using `# noqa `on this line

Reply:
I added an exclusion to the flake8 configuration